README.md

```diff
@@ -120,7 +120,7 @@ Now run your app and point prometheus to the defined metrics endpoint of your se
 You also can monitor a dependency event. Just call `collectDependencyTime` and pass the right params.
 
 ```js
-Monitor.collectDependencyTime(req, res, name, type)
+Monitor.collectDependencyTime(name, type, status, method, path, errorMessage, startTime)
```

Comment:
@KarineValenca, I'm worried the `startTime` argument could be troublesome for some devs. Maybe we should enhance our documentation to include the need to declare the `startTime` variable beforehand.

Reply:
Thanks, @abilioesteves. I improved the documentation to give instructions about the startTime variable.